django_ipam/api/views.py

```diff
@@ -0,0 +1,16 @@
+import swapper
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+Subnet = swapper.load_model('django_ipam', 'Subnet')
+
+
+@api_view(['GET'])
+def iprequest(request):
+    subnet_id = request.query_params.get('subnet_id')
+    subnet = Subnet.objects.get(pk=subnet_id)
```

Comment:
this will fail if the subnet doens't exist, write a test for this case and handle the failure with `get_object_or_404`, DRF should have a custom version of this function as well